src/org/scada_lts/mango/service/UsersProfileService.java

```diff
@@ -0,0 +1,122 @@
+package org.scada_lts.mango.service;
+
+import br.org.scadabr.vo.usersProfiles.UsersProfileVO;
+import com.serotonin.mango.vo.User;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.scada_lts.dao.DAO;
+import org.scada_lts.dao.UsersProfileDAO;
+import org.springframework.jdbc.core.BatchPreparedStatementSetter;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+public class UsersProfileService {
+
+    private static final Log LOG = LogFactory.getLog(UsersProfileService.class);
+    private static final String LIST_SIZE_IS_GREATER_THAN_1 = "List size is greater than 1.";
+
+    private UsersProfileDAO usersProfileDAO;
+
+    public UsersProfileService() {
+        this.usersProfileDAO = new UsersProfileDAO(DAO.getInstance().getJdbcTemp());
+    }
+
+    public UsersProfileService(UsersProfileDAO usersProfileDAO) {
+        this.usersProfileDAO = usersProfileDAO;
+    }
+
+    public Optional<UsersProfileVO> getProfileByUser(User user) {
+        return getProfileByUserId(user.getId());
+    }
+
+    public Optional<UsersProfileVO> getProfileByUserId2(int userId) {
+        List<UsersProfileVO> profiles = usersProfileDAO.selectUserProfileByUserId(userId);
+        if(profiles.isEmpty())
+            return Optional.empty();
+        if(profiles.size() > 1) {
+            LOG.warn(LIST_SIZE_IS_GREATER_THAN_1);
+            return Optional.empty();
+        }
+        return Optional.ofNullable(profiles.get(0));
+    }
```

Comment:
This method has no usages and is the same as method below. Also there are unused imports in this file.